.github/workflows/regression_test.yml

```diff
@@ -33,13 +33,19 @@ jobs:
             torch-spec: '--pre torch==2.7.0.dev20250122 --index-url https://download.pytorch.org/whl/nightly/cpu'
             gpu-arch-type: "cpu"
             gpu-arch-version: ""
-
+          - name: ROCM Nightly
+            runs-on: linux.rocm.gpu.torchao
+            torch-spec: '--pre torch --index-url https://download.pytorch.org/whl/nightly/rocm6.3'
+            gpu-arch-type: "rocm"
+            gpu-arch-version: "6.3"
     permissions:
       id-token: write
       contents: read
-    uses: pytorch/test-infra/.github/workflows/linux_job_v2.yml@main
+    uses: pytorch/test-infra/.github/workflows/linux_job_v2.yml@enable_linux_job_v2
     with:
+      enabled: ${{ matrix.gpu-arch-type != 'rocm' || (github.event_name == 'push' && startsWith(github.ref, 'refs/heads/main')) }}
```

Comment:
Hmm, instead of having this custom flag that is only used here.  I'm thinking it makes more sense to create a new workflow/job for ROCm instead.  We can have `if: ${{ matrix.gpu-arch-type != 'rocm' || (github.event_name == 'push' && startsWith(github.ref, 'refs/heads/main')) }}` there instead.

What are your thoughts on this? @atalman

Reply:
Hmm, I was trying to avoid the divergence and maintenance overhead that comes with having a duplicate workflow.

Reply:
Yes I think @huydhn suggestion is good. This is the format we normally use pretty much everywhere.

Reply:
Okay I created a separate workflow regression_test_rocm.yml in https://github.com/pytorch/ao/pull/1581/commits/a6958d79a4cdc627db28e5739d0da89bab185854